### PR TITLE
PRL-6677 & PRL-6678

### PR DESCRIPTION
**Pull request by RobL681**
### Change description

Retrieve access code for applicant and respondent via API

### JIRA link

[PRL-6677](https://tools.hmcts.net/jira/browse/PRL-6677)
[PRL-6678](https://tools.hmcts.net/jira/browse/PRL-6678)

**Before merging a pull request make sure that:**

- [x] Commits are meaningful and simple
- [x] All commits are squashed into a single commit
- [x] README and other documentation has been updated / added (if needed)

### CODE REVIEW

File: e2e/common/accessCodeHelper.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  private static async getAccessCode(ccdRef: string): Promise<CaseInvite[]> {
+    let caseInvites: CaseInvite[] = [];
+    const apiContextSystemUser: APIRequestContext = await request.newContext();
+    const apiContextS2SToken: APIRequestContext = await request.newContext();
+    const tokenSystemUserCreateCase = await getAccessToken(
+      "accessCode",
+      apiContextSystemUser,
+    );
+    const s2sToken = await getS2SToken(apiContextS2SToken, "prl_cos_api");
```
Comment (JayAshton) [inline e2e/common/accessCodeHelper.ts:41]: Should this be a secret or if not, should it be declared separately?

**Review by JayAshton — APPROVED** [review]

**Review by pavihmcts — APPROVED** [review]

File: e2e/common/accessCodeHelper.ts (hunk 1 above)
Comment (RobL681) [reply e2e/common/accessCodeHelper.ts:41]: Are you thinking the "prl_cos_api" should be a secret?

Comment (JayAshton) [reply e2e/common/accessCodeHelper.ts:41]: Not sure if it's sensitive or not, guessing not as it doesn't look to be a secret already in the service
But maybe it should be a class property

Comment (RobL681) [reply e2e/common/accessCodeHelper.ts]: Yeah I don't think it needs to be a secret looking at other repos. I've just moved it to a variable lmk if you're happy

**Review by JayAshton — APPROVED** [review]

**Review by abigailsmith97 — APPROVED** [review]

**Pull request merged**